### DISCUSSION TITLE
Section 3: LaTeX Error: File `scrartcl.cls' not found

### DISCUSSION THREAD

**Discussion by truth2012** [Cyfrin/security-and-auditing-full-course-s23#127] [Q&A] [answered]
I have been trying to convert my report.md into report.pdf but i keep coming across the below error i have tried every solution Ai has given me, reinstalled latex but same result. Can someone help me please
This is the error message below;
Error producing PDF.
! LaTeX Error: File `scrartcl.cls' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: cls)

Enter file name: 
! Emergency stop.
<read *> 
         
l.52 \usepackage

**Comment by allwin199** [comment]
@truth2012 I have made a discussion regarding alternative ways to [Create PDF](https://github.com/Cyfrin/security-and-auditing-full-course-s23/discussions/126).

**Comment by alymurtazamemon** [comment]
@truth2012 Did you check this [link](https://github.com/overleaf/overleaf/issues/876)?

**Comment by alymurtazamemon** [comment, marked as answer]
@truth2012 You should run this command to install all required packages at once.

```bash
sudo tlmgr install adjustbox babel-german background bidi collectbox csquotes everypage filehook footmisc footnotebackref framed fvextra letltxmacro ly1 mdframed mweights needspace pagecolor sourcecodepro sourcesanspro titling ucharcat ulem unicode-math upquote xecjk xurl zref
```

If you still see missing package issue then you can install that using this command;

```bash
sudo tlmgr install package-name
```
For example in your case it should be;
```bash
sudo tlmgr install scrartcl
```

I am using a Mac, if you still have problems, please check out the Pandoc website for information related to your operating system.
